static/scripts/main.js

```diff
@@ -6,23 +6,39 @@ window.onload = function () {
     if(hash) {
       hash.scrollIntoView()
     }
+    $('[data-md-sticky-header]').headroom({
+      offset: UnpinOffset,
+      tolerance: 0
+    });
 }
 
-var CurrentScroll = 0;
+var SearchbarTop = 61;
+var UnpinOffset = 400;
+var PrevScrollTop = 0;
 
 $(window).scroll(function() {
-  var NextScroll = $(this).scrollTop();
-  if (NextScroll < CurrentScroll && NextScroll > 67){
-     //upscroll
-     $('[data-md-sticky-header]').addClass('sticky');
+  function handleScrollUp() {
+    if (ScrollTop > SearchbarTop && ScrollTop <= UnpinOffset)
+      $('[data-md-sticky-header]').addClass('sticky');
```

Comment:
This element is being queried on each scroll event. Can we save a reference outside the scroll event handler and reuse here?